### ISSUE TITLE
Use Github Pages

### ISSUE THREAD

**Issue by dsevillamartin** [HaemimontGames/SurvivingMars#2]
I think it’d benefit the repository and viewers if Github Pages was used, that way the HTML documents in Docs/ could be viewed online (not just the source code) more easily.

Github Pages let’s you choose between the `master` and `gh-pages ` branches and the `docs/` folder in master. Not sure if the docs folder would be case insensitive, but I’d imagine so.

EDIT: Though the files are just Markdown files with the HTML extension... not sure if github Pages supports markdown itself, but it should be fine in the html file.

**Comment by ivanassen** [comment]
Yeah, this is the plan, we just didn't have the time for the initial release.

**Comment by caitlynrw** [comment]
I can work on a GitHub Pages Documentation/Wiki thing if you need
